src/js/api/api.js

```diff
@@ -177,11 +177,8 @@ define([
         };
 
         this.load = function (toLoad) {
-            var plugin = this.getPlugin('vast') || this.getPlugin('googima') || this.getPlugin('freewheel');
-            if (plugin) {
-                _controller._model.set('preInstreamState', 'instream-idle');
-                plugin.destroy();
-            }
+            _controller._model.set('preInstreamState', 'instream-idle');
+            this.trigger(events.JWPLAYER_LOAD_PLAYLIST, {});
```

Comment:
Load may load a playlist or set the current item. See controller `_load`.
```js
                switch (typeof item) {
                    case 'string':
                        _loadPlaylist(item);
                        break;
                    case 'object':
                        var success = _setPlaylist(item);
                        if (success) {
                            _setItem(0);
                        }
                        break;
                    case 'number':
                        _setItem(item);
                        break;
                }
```
I don't think this event is correct where you've placed it. It would only make sense if item was a string, but we may still need to do something in the case that it's an object.